inc/com/centreon/handle_manager_posix.hh

```diff
@@ -0,0 +1,74 @@
+/*
```

Comment:
why not renaming this file handle_manager.hh, and remove the inderection ?

Reply:
Yes, it was planned :smile:

Reply:
Done.